URA-695/run_sentieon-tnbam.py

```
@@ -0,0 +1,102 @@
+set_job_ids = []
+for sample in bamfile_list:
+    job_ids = {}
+    job_ids['Name'] = sample['Name']
+    
+    job_input = {
+        'gatk_resource_bundle': 
+            dx.dxlink('file-F3zvKp84fXX8qJx43zZXP395',
+                      project_id='project-F3zqGV04fXX5j7566869fjFq'),
+        'genome_fastagz': dx.dxlink('file-Gb757784XGyY3FPvkPQ74K9z'),
+        'run_bqsr': True,
+        'somatic_algo': 'TNhaplotyper2'
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)